go.mod

```diff
@@ -571,6 +571,8 @@ require (
 	github.com/aquasecurity/trivy v0.0.0-00010101000000-000000000000
 	github.com/cloudfoundry-community/go-cfclient/v2 v2.0.1-0.20230503155151-3d15366c5820
 	github.com/godror/godror v0.37.0
+	github.com/jellydator/ttlcache/v2 v2.11.1
```

Comment:
I think you can now run `go mod tidy` to clean up the ttlcache dependency that's unused.

Reply:
done, thanks!